org/mozilla/jss/tests/all.pl

```diff
@@ -207,7 +217,7 @@ sub setup_vars {
     if ($osname =~ /Darwin/) {
         $java = "$ENV{JAVA_HOME}/bin/java";
```

Comment:
We could perhaps refactor some of this logic as well.

Reply:
Perhaps but notice that setup_vars on all.pl is quite different from the one in build_java.pl